google-cloud-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITBucketSnippets.java

```diff
@@ -133,4 +139,47 @@ public void testBucket() throws InterruptedException {
     thrown.expect(StorageException.class);
     assertTrue(bucketSnippets.delete());
   }
+
+  @Test
+  public void testListBucketIamMembers() {
+    // Test an added Bucket-level IAM member is listed 
+    Policy policy = storage.getIamPolicy(BUCKET);
+    policy = storage.setIamPolicy(BUCKET,
+        policy.toBuilder().removeRole(StorageRoles.admin()).build());
+    assertNull(policy.getBindings().get(StorageRoles.admin()));
+    policy = storage.setIamPolicy(BUCKET, policy.toBuilder().addIdentity(StorageRoles.admin(),
+        Identity.user(USER_EMAIL)).build());
+    assertTrue(policy.getBindings().get(StorageRoles.admin()).contains(Identity.user(USER_EMAIL)));
+    Policy snippetPolicy = bucketIamSnippets.listBucketIamMembers(BUCKET);
+    assertTrue(snippetPolicy.getBindings().get(StorageRoles.admin()).
```

Comment:
This has a chance of failure
@tswast @garrettjonesgoogle I don't have a suggestion on how Frank might fix this assert.  Suggestions?
@frankyn Unless Tim / Garrett suggest something creative, I'd start with a warning comment above the assert, and if it fires a lot, we should just comment out this line.